variables.tf

```diff
@@ -35,5 +35,6 @@ variable "subnet_assignments" {
     wfrancis        = "10.59.0.0/16"
     trobbins        = "10.60.0.0/16"
     aadriasola      = "10.61.0.0/16"
+    dev1            = "10.62.0.0/16"
```

Comment:
for consistency, since this is intended to be a shared grid, can you move it to a lower range like maybe after qa?